app/controllers/papers_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
   # GET /papers/new
   def new
     @paper = Paper.new
+    @authors = 5.times {@paper.authors.build}
```

Comment:
yea, this will add 5 instances of the authors text field.  I'm also wondering how we want to handle title.  For now I'm ignoring it because I feel like its not essential.  It might even be easier later on if we delimit titles from the names later on from name.